File: 326. Power of Three.js
/**
 * @param {number} n
 * @return {boolean}
 */
var isPowerOfThree = function(n) {
    if (n <= 0) return false;
    var temp = Math.log10(n) / Math.log10(3);
    if (Math.round(temp) == temp) return true;
    return false;
};

File: 326. Power of Three.py
class Solution(object):
    def isPowerOfThree(self, n):
        """
        :type n: int
        :rtype: bool
        """
        if n <= 0:
            return False
        temp = math.log10(n) / float(math.log10(3))
        return True if int(temp) == temp else False
